docs/articles/documentation/test-api/executing-code-in-the-browser/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: docs
+title: Executing Code in the Browser
+permalink: /documentation/test-api/executing-code-in-the-browser/
+---
+# Executing Code in the Browser
```

Comment:
I have a strong feeling that we should extract selector-related stuff from here. Yes, they have look-a-like machinery. But selectors are really different from client functions. And I will not search selector-related stuff in `Executing Code in the Browser` section